pending/tg.js

```diff
@@ -596,154 +596,182 @@ server.config.lives Mga Buhay
-stats.assists Mga tumutulong
-stats.nukes Nukes
-stats.melee Melee
-stats.fistkills Mga Beatdown
-stats.thrownkills Bullseyes
+stats.assists Mga pagtulong at gabay
+stats.nukes Mga Nuke
+stats.melee Melee Sinaksak
+stats.fistkills Mga Binugbog
+stats.thrownkills Mga natamaan ng panaksak
 stats.headshots Mga headshot
 stats.wallbangs Mga Wallbang
-stats.sprays Nag-spray ng Inilagay na spray
+stats.sprays Inilagay na spray
 windows.host.default.settings Ang pagpapagana ng Default na Mga Setting ay nagbibigay-daan para sa pagtatago ng Player Progress at Highscores
 windows.host.default.commands * Ang Mga Command ng Host ay Hindi Pinapagana sa mode na ito: / noclip / god / smite atbp
 windows.host.rewards Pinipilit ng pagpapagana ng KR Rewards ang Server na gumamit ng Default na Mga Setting
-windows.store.wheels.sub Subukan ang iyong kapalaran at paikutin para sa isang pagkakataon upang manalo ng ilang mga bagong balat!
-Windows.store.currency * Ang mga presyo sa USD ay maaaring mai-convert sa iyong lokal na pera sa isang rate ng palitan na tinukoy ng provider ng pagbabayad o bangko
+windows.store.wheels.sub Subukan ang iyong kapalaran at paikutin para sa isang pagkakataon upang manalo ng ilang mga bagong disenyo!
+windows.store.currency * Ang mga presyo sa USD ay maaaring mai-convert sa iyong lokal na pera sa isang rate ng palitan na tinukoy ng provider ng pagbabayad o bangko
 windows.store.creator Isaalang-alang ang Suportahan ang iyong paboritong Tagalikha ng Nilalaman sa pamamagitan ng paggamit ng isang Creator Code
-windows.chall.customs Ang mga hamon ay hindi nakukumpleto sa Pasadyang Mga Pagtutugma
-twitch.link Sigurado ka bang nais mong mai-link ang isang twitch account?
+windows.chall.customs Ang mga hamon ay hindi nakukumpleto sa mga kustom na laro
+twitch.link Sigurado ka bang nais mong i-link ang twitch account?
 twitch.unlink Sigurado ka bang nais mong i-unlink ang iyong twitch account?
-windows.clan.leave Are you sure you want to leave the clan?
-windows.clan.delete Are you sure you want to delete the clan?
-windows.clan.kick Sigurado ka bang nais mong sipa {0} mula sa angkan?
-windows.store.premium.purchase Sigurado ka bang nais mong bumili {0} araw ng Premium?
-windows.playerlist.ban Sigurado ka bang nais mong pagbawal {0} mula sa larong ito?
+windows.clan.leave Sigurado ka bang nais mong umalis sa hukbo/samahan ninyo?
+windows.clan.delete Sigurado ka bang nais mong tanggalin ang hukbo/samahan?
+windows.clan.kick Sigurado ka bang nais mong i-kick si {0} at tanggalin sa samahan?
+windows.store.premium.purchase Sigurado ka bang nais mong bumili ng {0} araw ng Premium?
+windows.playerlist.ban Sigurado ka bang nais mong ipagbawal si {0} mula sa larong ito?
 windows.playerlist.flag Sigurado ka bang nais mong I-flag {0}?
 windows.playerlist.unflag Sigurado ka bang nais mong mag-Unflag {0}?
-windows.playerlist.punish Sigurado ka bang nais mong kumilos sa {0}?
+windows.playerlist.punish Sigurado ka bang nais mong parusahin ang manlalaro {0}?
```

Comment:
can you double check this one please? `{0}` was missing, so I re-added it.

Reply:
yes its fine.